tests/snapd

```diff
@@ -36,6 +25,8 @@ cleanup() {
         lxc storage delete "${storage}" || REBOOT=1
     done
 
+    userdel -r ${test_user} 2>/dev/null || true
```

Comment:
Could you drop that `cleanup()` function and instead do something similar to https://github.com/canonical/lxd-ci/blob/main/tests/efi-vars-editor-vm#L76-L86 while keeping the `userdel` part, ofc.

There is already a `cleanup()` function in `bin/helpers` so this local one is shadowing it anyway.

Reply:
Done, thanks for the reference link!